packages/@react-aria/utils/src/useGlobalListeners.ts

```diff
@@ -21,7 +21,11 @@ interface GlobalListeners {
 }
 
 export function useGlobalListeners(): GlobalListeners {
-  let globalListeners = useRef(new Map());
+  let globalListeners = useRef<Map<any, any>>();
```

Comment:
this looks the same as it was previously, I don't see anything in the code which sets globalListeners to null/undefined/etc, what use case did you run into where globalListeners.current was falsy?